app/backend/src/clj/teet/road/road_query.clj

```diff
@@ -152,48 +152,63 @@
                p points))))
   (println "part " (:start-m road-part) (:end-m road-part)))
 
-
-(defn- wfs-request [wfs-url query-params]
-  (let [{:keys [error body]}
-        @(client/get wfs-url
-                     {:connect-timeout 10000
-                      :query-params (merge
-                                     {:SERVICE "WFS"
-                                      :REQUEST "GetFeature"
-                                      :VERSION "1.1.0"
-                                      :TYPENAME "ms:teeosa"
-                                      :SRSNAME "urn:ogc:def:crs:EPSG::3301"}
-                                     query-params)
-                      :as :stream})]
-    (if error
-      (throw (ex-info "Unable to fetch road parts from WFS"
-                      {:status 500
-                       :error :wfs-request-failed
-                       :wfs-url wfs-url
-                       :query-params query-params}
-                      error))
-      (-> body xml/parse zip/xml-zip
-          read-feature-collection))))
-
-(defn fetch-road-parts [{wfs-url :wfs-url} road-nr carriageway-nr]
-  (wfs-request wfs-url {:FILTER (query-by-road-and-carriageway road-nr carriageway-nr)}))
-
-(defn fetch-road-parts-by-coordinate [{wfs-url :wfs-url} coordinate distance-m]
-  (wfs-request wfs-url {:FILTER (query-by-coordinate coordinate distance-m)}))
+(defn- cached [cache-atom cache-key fetch-fn]
```

Comment:
Is this for development only? Since the `cache-options` is a commented atom.

Reply:
Yes the cache options is for train development, you can fetch the items while in good network and have it cached.

The cache-atom is also used in THK import when updating multiple projects' geometries.

Reply:
Should rename and document the dev-only parts so that it's clear they're for dev use.

Reply:
Caching is not a dev-only part, but I will document the cache-options in the queries